apps/dashboard/config/configuration_singleton.rb

```diff
@@ -20,6 +20,8 @@
 # ConfigurationSingleton and defining it as a class method on Configuration.
 #
 class ConfigurationSingleton
+  require_relative '../app/models/concerns/bool_reader.rb'
```

Comment:
This is a bit of a nit-pick, but can we move this up to the other requires? I think I just want uniformity here.